src/index.html

```diff
@@ -7,6 +7,261 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <a class="nav__item is-active">Apple</a>
+          <a class="nav__item">Samsung</a>
+          <a class="nav__item">Smartphones</a>
+          <a class="nav__item" data-qa="nav-hover">laptops & computers</a>
+          <a class="nav__item">gadgets</a>
+          <a class="nav__item">tablets</a>
+          <a class="nav__item">photo</a>
+          <a class="nav__item">video</a>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="cards">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="card img"
+            class="card__image"
+          >
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <p class="card__description">Product code: 195434</p>
+          <div class="card__rating">
+            <div class="stars stars--4">
```

Comment:
```suggestion
          <div class="card__rating">
            <div class="card__stars stars stars--4">
```

![image](https://github.com/mate-academy/layout_catalog/assets/59209337/e391ec38-9823-40e8-a930-7c0b960dc5ac)